raiden/api/webui/static/endpointConfig.js

```diff
@@ -1,7 +1,7 @@
-const backendUrl='http://localhost:5002'; 
-const nodeAddress = '0x62Aa1173561C4E92876B3D073B9767A2d07496f8'; 
-const rnsDomain = null 
-const chainEndpoint = 'http://localhost:4444'; 
+const backendUrl='http://localhost:5002';
+const nodeAddress = '0x62Aa1173561C4E92876B3D073B9767A2d07496f8';
+const rnsDomain = null
```

Comment:
it's good that you have reverted the changes here, but i would still add `;` at the end of line `3`